subprojects/docs/src/docs/userguide/workingWithFiles.adoc

```diff
@@ -355,7 +831,7 @@ For example, to make a `Zip` task reproducible you need to set api:org.gradle.ap
 Often you will want to publish an archive, so that it is usable from another project. This process is described in <<artifact_management>>
 
 [[sec:properties_files]]
-=== Properties files
+=== Java properties files
```

Comment:
This subsection doesn't seem to fit in very well. Is there another place it belongs?

Reply:
Completely agree, but left it in for now. Should probably go in the Java Plugin chapter.

Reply:
Would you be opposed to moving it in this PR?

Reply:
Done